packages/yarnpkg-cli/sources/main.ts

```diff
@@ -57,13 +57,11 @@ export async function main({binaryVersion, pluginConfiguration}: {binaryVersion:
 
   async function exec(cli: Cli<CommandContext>): Promise<void> {
     // Non-exhaustive known requirements:
-    // - 10.16+ for Brotli support on `plugin-compat`
-    // - 10.17+ to silence `got` warning on `dns.promises`
     // - 14.0 and 14.1 empty http responses - https://github.com/sindresorhus/got/issues/1496
     // - 14.10.0 broken streams - https://github.com/nodejs/node/pull/34035 (fix: https://github.com/nodejs/node/commit/0f94c6b4e4)
 
     const version = process.versions.node;
-    const range = `>=10.17 <14 || 14.2 - 14.9 || >14.10.0`;
+    const range = `>=12 <14 || 14.2 - 14.9 || >14.10.0`;
```

Comment:
Should we make the minimum the LTS version?

Reply:
Forgot to update - perhaps we can leave it at 12 until we get first reports that it really doesn't work on some versions.